libcontainer/criu_disabled_linux.go

```diff
@@ -0,0 +1,15 @@
+//go:build runc_nocr
+
+package libcontainer
+
+import "errors"
+
+var ErrNoCR = errors.New("checkpoint/restore support not compiled in")
```

Comment:
Since this is going to be a user-visible error, we should probably make the error a little more friendly, something like:

```suggestion
var ErrNoCR = errors.New("this runc binary has not been compiled with checkpoint/restore support enabled (runc_nocr)")
```

Reply:
done